proto/kvrpcpb.proto

```diff
@@ -84,6 +84,7 @@ message Context {
     bool scan_detail = 11; // true means return scan cf's detail
     bool replica_read = 12;
     repeated uint64 resolved_locks = 13;
+    uint64 end_point_request_max_handle_duration = 14;
```

Comment:
prefer to simply call it "max_handle_duration_ms" or something like that

Reply:
@breeswish The coprocessor execution time contains `wait_time` and `handle_time`, I think it's better to `max_execution_duration_ms` is more appropriate.

Reply:
Should I change it? `handle_time` is a bool though. 🤔

Reply:
@lonng Nice catch. `max_execution_duration_ms` +1

@liufuyang Actually the field `handle_time` means whether or not the handle time should be recorded.